presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/ElasticsearchQueryRunner.java

```diff
@@ -38,6 +38,14 @@
 import static java.util.Locale.ENGLISH;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
+/**
+ * Runs a Presto server with the Elasticsearch connector configured to talk
+ * to an external Elasticsearch instance.
+ *
+ * To start Elasticsearch:
+ *
+ * docker run -p 9200:9200 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.6.2
```

Comment:
This should be on `main()` method.

The class in general, does not expect ES to be running on a prticular port (the createQueryRunner method takes hostandport for this)